src/tests/components/UserProfile.test.js

```diff
@@ -2,35 +2,79 @@ import React from 'react';
 import Enzyme, { shallow, mount } from 'enzyme';
 import Adapter from 'enzyme-adapter-react-16';
 import configureStore from 'redux-mock-store';
-import { updateProfileInfo } from '../mockData/profile';
-import { profile } from '../mockData/profile';
 import thunk from 'redux-thunk';
+import { profile, updateProfileInfo } from '../mockData/profile';
 import { UserProfileComponent, mapDispatchToProps } from '../../components/user/UserProfile';
 import UserStats from '../../components/user/UserStats';
+import { FollowUnfollow } from '../../components/user/FollowUnfollow';
+import FollowUnfollowComponent from '../../components/user/FollowUnfollow';
+import * as testData from '../mockData/profile';
+import AuthorProfileComponent, { AuthorProfile } from '../../components/user/AuthorProfile';
+import { Provider } from 'react-redux';
 
 Enzyme.configure({ adapter: new Adapter() });
 
 const middleware = [thunk];
 const mockStore = configureStore(middleware);
 const articles = {
-  personalArticles : []
+  personalArticles: [],
 };
 const store = mockStore({ articles });
 
 store.articles = articles;
 const fetchProfile = jest.fn();
 
-const clickFn = jest.fn();
-const profileData = {
-  name: 'brian',
+const props = {
+  authorProfile: testData.authorProfile,
+  match: {
+    params: {
+      username: 'brina',
+    },
+  },
 };
 
+describe('FollowUnfollow', () => {
+  const component = shallow(<FollowUnfollowComponent profile={testData.authorProfile} />);
+  const wrapperInstance = component.instance();
+
+  it('renders a single profile page', () => {
+    const component = shallow(<FollowUnfollow profile={testData.authorProfile} />);
+    expect(component).toHaveLength(1);
+    const wrapperInstance = component.instance();
+  });
+  it('renders a single profile page', () => {
+    const component = shallow(<FollowUnfollowComponent profile={testData.authorProfile} />);
+    expect(component).toHaveLength(1);
+    const wrapperInstance = component.instance();
+  });
+  // it("should properly show registration form when the span is clicked", () => {
+  //   const createSpy = toSpy => jest.spyOn(wrapperInstance, toSpy);
+  //   const onClick = createSpy("onClick");
+  //   wrapperInstance.forceUpdate();
+  //   const button = wrapper.find(".btn-one").at(1);
+  //   button.simulate("click");
+  //   expect(onClick).toHaveBeenCalled();
+  // });
```

Comment:
@BrianSerem, remove this commented out code.

Reply:
I have removed these commented lines of code.